### ISSUE TITLE
Could you please publish the specific hyper-parameters for each dataset?

### ISSUE THREAD

**Issue by yunchong-song** [Wu-Junran/SEP#1]
Hi, this is a great work, however for hyper-parameters, you only mentioned:

> For model configuration, the learning rate is set to 5 × 10−4, the hidden size is set ∈ {64, 128}, the batch size is set ∈ {32, 128}, weight decay is set to 1 × 10−4, and dropout rate is set ∈ {0, 0.5}.

in the appendix of SEP paper, and I didn't find specific hyper-parameters for each dataset in your code (except for example dataset IMDB-B), could you please publish it? It's important for us to follow your work.

**Comment by Wu-Junran** [comment]
The hyperparameters chosen in this work are listed as follows:

Dataset | Hidden Dim | Batch Size | Dropout | Lr Schedule
:---:|:---:|:---:|:---:|:---:|
IMDBBINARY	|64	|128 |	0	|False
IMDBMULTI |64	|128	|0.5	|True
COLLAB	|128	|128|	0|	False
MUTAG	|64	|128|	0.5|	True
PROTEINS	|64	|128	|0	|False
NCI1 |64|128	|0.5	|True
DD |64	|128	|0	|True

Thanks for your interest and wish you work well.

**Comment by yunchong-song** [comment]
> The hyperparameters chosen in this work are listed as follows:
> 
> Dataset	Hidden Dim	Batch Size	Dropout	Lr Schedule
> IMDBBINARY	64	128	0	False
> IMDBMULTI	64	128	0.5	True
> COLLAB	128	128	0	False
> MUTAG	64	128	0.5	True
> PROTEINS	64	128	0	False
> NCI1	64	128	0.5	True
> DD	64	128	0	True
> Thanks for your interest and wish you work well.

Thank you for your reply, got it.
